### PR TITLE
Update team-leadership.md

### PR DESCRIPTION
**Pull request by MelSumner**
Proposing a new structure for our team meetings.

### CODE REVIEW

**Comment by jenweber** [comment]
I think this is a great move to be more inclusive of varied interests, timezones, work commitments, life commitments, etc. I would look forward to facilitating a group on the off-weeks that is focused solely on writing reference materials and tutorials.

**Comment by MelSumner** [comment]
> This LGTM! I plan to continue working on Inspector, but would like to get more involved with general learning team stuff again as well, so I guess I would fall under `infrastructure & tooling` and then some various other things.

I think I include inspector in "tooling" but we can explicitly define that if it's useful 👍

**Comment by amyrlam** [comment]
as for me: stay with ember times and also move into guides/api docs/tutorials

File: team-leadership.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 
-- set up the agenda on Dropbox, ideally by the Monday of the week of the meeting
-  - Create a document using the `Learn Team Agenda 2019-MM-DD` template
-  - Add link to previous week's meeting
-  - Update the "Every Timezone" link
-- post the link in the #core-learning channel description. We don't share the link in public chat channels. Once the meeting has started, we share link to the video call in the public channel. Then we share the agenda with the attendees via the video chat interface.
-- act as the meeting facilitator (leading the discussion, ensuring that we stay on track with the agenda & conversation)
-- extract the public notes and submit them to the [GitHub repository for core team notes](https://github.com/emberjs/core-notes)
-- synthesize the to-do items into issues in the relevant GitHub repository for tracking
+The day and time of the meeting will stay the same- but folks will break out into their areas of responsibility (AOR). Once a month, all AORs will meet for an all-hands meeting so that we all stay updated on all facets of the learning team.
```
Comment (mansona) [inline team-leadership.md:11]: I am behind this whole plan, but I think we need a bit more detail about this plan of having the AOR meetings at the same time 🤔 for example, there are some cases where some Learning Team members may be working across two AORs but if the meetings are happening at 15:30 UTC then they would not be able to attend both meetings.

Is the intention that they should not be spreading themselves too thin?

Comment (MelSumner) [reply team-leadership.md:11]: @mansona yes exactly. We are trying to help ourselves by preventing this from happening.

Comment (mansona) [reply team-leadership.md:11]: I guess I'm concerned that it makes it harder for other teams to come to join a particular meeting 🤔 like for example @rwjblue has said that he wanted to join more meetings. 

Would the intention be that he only join the once-a-month meeting then? (feel free to speak for yourself on this one @rwjblue 😂 I'm just trying to understand how it's **intended** to work)

Comment (MelSumner) [reply team-leadership.md:11]: With love and respect to @rwjblue, it shouldn't be a deciding factor in how the learning team works. Some of our team is spread too thin, and that's actively preventing new growth. 

I think by getting us to choose one AOR and cultivate that AOR and the community members that want to participate in that particular facet of the learning team, we will enable ourselves to move forward. Things have been super slow this past year and there have been significant bottlenecks in our process, and this move would alleviate that.

Comment (mansona) [reply team-leadership.md]: The comment about @rwjblue was only as a reference, I agree that we shouldn't be using **specific** people outside of the team as a reason for not going ahead.

I know that we discussed this last week and I'm ok to approve this PR if we add a comment about allowing for moving meetings so they don't clash on a case-by-case basis 👍

Comment (mansona) [reply team-leadership.md]: The comment about @rwjblue was only as a reference, I agree that we shouldn't be using **specific** people outside of the team as a reason for not going ahead.

I know that we discussed this last week and I'm ok to approve this PR if we add a comment about allowing for moving meetings so they don't clash on a case-by-case basis 👍

**Pull request closed without merging**